tests/nnx/partitioning_test.py

```diff
@@ -150,9 +150,6 @@ def test_get_paritition(self):
       d=5.0,
     )
 
-    # test Variables not shared
-    self.assertIsNot(vars(m.a)[0], vars(m)['b'])
```

Comment:
Why remove this?

Reply:
doesn't make sense here, probably copied from another test in the past